File: typescript/apps/app2/clsdir/person.ts
export class Person {
    name: string;
    age: number;
    address: string;

    constructor(name: string, age: number, address: string) {
        this.name = name;
        this.age = age;
        this.address = address;
    }

    private sayName(): void {
        console.log("My name is " + this.name + "!");
    }

    public getName(): string {
        this.sayName();
        return this.name;
    }
}

File: rust/apps/app2/rustapp/src/cls/person.rs
#[derive(Debug)]
pub struct Person {
    pub name: String,
    pub age: i8,
    pub country: String
}

impl Person {
    pub fn new(name:String, age: i8, country: String) -> Person {
        Person { name: name
                 , age: age
                 , country: country}
    }
}